Documentation/observability/metrics.rst

```diff
@@ -1115,6 +1115,7 @@ Name                                     Labels
 ``kvstore_sync_errors_total``            ``scope``, ``source_cluster``                Number of times synchronization to the kvstore failed
 ``kvstore_sync_queue_size``              ``scope``, ``source_cluster``                Number of elements queued for synchronization in the kvstore
 ``kvstore_initial_sync_completed``       ``scope``, ``source_cluster``, ``action``    Whether the initial synchronization from/to the kvstore has completed
+``kvstore_initial_sync_time_seconds``                                                 Duration in seconds of the initial sync to the kvstore for local cluster resources
```

Comment:
Just double-checking to be sure: No related labels for these new metrics?

Reply:
Renamed to `bootstrap_seconds` and added a `source_cluster` label. Decided against adding `outcome` to surface timeouts. It added complexity and that info is already available  via the other metrics.